src/pages/guide/integration/data-fixtures-guide.md

```diff
@@ -90,190 +93,62 @@ Bootstrap::getObjectManager()
 
 ### 2. Parametrized data fixtures (Recommended)
 
-These are PHP classes that implement `DataFixtureInterface` or `RevertibleDataFixtureInterface`. They support **powerful parametrization** and can be used with **both DocBlock annotations and PHP Attributes**.
+These are PHP classes that implement `DataFixtureInterface` or `RevertibleDataFixtureInterface`. They support **powerful parametrization** and **require PHP Attributes** (`#[DataFixture()]`).
+
+<InlineAlert variant="warning" slots="text" />
+
+Parametrized data fixtures **cannot** be used with DocBlock annotations. Only PHP Attributes support parametrization.
 
 #### With PHP Attributes (Best option)
 
 ```php
 #[
-    DataFixture(ProductFixture::class, ['sku' => 'custom-sku', 'price' => 99.99], 'product')
+    DataFixture(ProductFixture::class, ['price' => 10.00], 'product')
 ]
 public function testProductExists(): void
 {
     $fixtures = DataFixtureStorageManager::getStorage();
     $product = $fixtures->get('product');
-    // Test with customized product data
-}
-```
-
-#### With DocBlock Annotations (Parametrized syntax)
-
-DocBlock annotations support parametrized data fixtures using the `with:` and `as:` syntax:
-
-```php
-/**
- * @magentoDataFixture \Magento\Catalog\Test\Fixture\Product with:{"sku": "simple-product", "price": 10} as:product
- */
-public function testProductExists(): void
-{
-    $fixtures = DataFixtureStorageManager::getStorage();
-    $product = $fixtures->get('product');
-    // Test with customized product data
+    // Only override values that matter for your test
+    // SKU is auto-generated with a unique value
 }
 ```
 
-**Syntax:**
-- `with:{"key": "value"}` - Pass parameters as JSON object
-- `as:alias` - Assign an alias to reference the fixture data
-
 ## Parameterized data fixtures
 
 The power of modern parametrized data fixtures lies in their ability to be customized - you can configure the data they create without writing new fixture classes.
 
-<InlineAlert variant="success" slots="text" />
+<InlineAlert variant="warning" slots="text" />
 
-Parametrized data fixtures work with **both** DocBlock annotations (using `with:` and `as:` syntax) and PHP Attributes. PHP Attributes offer better IDE support and type safety.
+Parametrized data fixtures **require PHP Attributes**. They cannot be used with DocBlock annotations (`@magentoDataFixture`).
 
 ### With PHP Attributes
```

Comment:
this header is no longer necessary as this is the only way to declare a datafixture